RecoLocalCalo/Configuration/python/ecalLocalRecoSequence_cff.py

```diff
@@ -23,20 +23,29 @@
 #ecalUncalibRecHitSequence = cms.Sequence(ecalGlobalUncalibRecHit*
 #                                         ecalDetIdToBeRecovered)
 
-ecalUncalibRecHitTask = cms.Task(ecalMultiFitUncalibRecHit,
-                                        ecalDetIdToBeRecovered)
+ecalUncalibRecHitTask = cms.Task(
+    ecalMultiFitUncalibRecHitTask,
+    ecalDetIdToBeRecovered)
 
-ecalRecHitTask        = cms.Task(ecalRecHit,
-                                         ecalCompactTrigPrim,
-                                         ecalTPSkim,
-                                         ecalPreshowerRecHit)
+ecalRecHitTask = cms.Task(
+    ecalRecHit,
+    ecalCompactTrigPrim,
+    ecalTPSkim,
+    ecalPreshowerRecHit)
```

Comment:
wouldn't it be more straightforward to define ```ecalRecHitNoTPTask```  with ```ecalRecHit, ecalPreshowerRecHit``` so that it can be directly used in both ecalRecHitTask and ecalOnlyLocalRecoTask?